x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/saml/SamlMetadataCommand.java

```diff
@@ -110,22 +110,22 @@ public SamlMetadataCommand(CheckedFunction<Environment, KeyStoreWrapper, Excepti
         attributeSpec = parser.accepts("attribute", "additional SAML attributes to request").withRequiredArg();
         orgNameSpec = parser.accepts("organisation-name", "the name of the organisation operating this service").withRequiredArg();
         orgDisplayNameSpec = parser.accepts("organisation-display-name", "the display-name of the organisation operating this service")
-                .availableIf(orgNameSpec).withRequiredArg();
+            .availableIf(orgNameSpec).withRequiredArg();
```

Comment:
Can we undo all the whitespace changes here?

Reply:
fixed